Resources/doc/index.rst

```diff
@@ -80,25 +110,8 @@ the ``status`` command:
 
     $ php bin/console doctrine:migrations:status
 
-     == Configuration
-
-        >> Name:                                               Application Migrations
-        >> Database Driver:                                    pdo_mysql
-        >> Database Host:                                      127.0.0.1
-        >> Database Name:                                      symfony_migrations
-        >> Configuration Source:                               manually configured
-        >> Version Table Name:                                 migration_versions
-        >> Version Column Name:                                version
-        >> Migrations Namespace:                               DoctrineMigrations
-        >> Migrations Directory:                               /path/to/project/app/Migrations
-        >> Previous Version:                                   Already at first version
-        >> Current Version:                                    0
-        >> Next Version:                                       Already at latest version
-        >> Latest Version:                                     0
-        >> Executed Migrations:                                0
-        >> Executed Unavailable Migrations:                    0
-        >> Available Migrations:                               0
-        >> New Migrations:                                     0
+This command will show you generic information about the migration status, as how many migrations have been
+already executed, which still needs to run, the database in use and other similar informations.
```

Comment:
Since we also say that we show generic information about the migration status, we may as well omit the last part of the enumeration:

```suggestion
already executed, which still needs to run, and the database in use.
```